src/app/state/wallet/types.ts

```diff
@@ -43,6 +42,6 @@ export interface AddWalletPayload extends Wallet {
 /* --- STATE --- */
 export interface WalletState {
   isOpen: boolean
-  selectedWallet?: number
-  wallets: { [id: number]: Wallet }
+  selectedWallet?: string
+  wallets: { [address: string]: Wallet }
```

Comment:
Instead of using  `{ [address: string]: Wallet }` I find the `Record<string, Wallet>` form easier to read.

Reply:
I think the key name `address` is helpful here. Too many things are a string

Reply:
That's true. In fact it's so true that I would consider adding `export type Address = string` somewhere, and then consistently using `Address` instead of `string` wherever we are talking about addresses. (But that's a story of a different PR.)

Reply:
Yeah :D would we consistently use it tho? maybe :D 

If we add it, we should alias it like StringifiedBigInt, or make it branded
https://github.com/oasisprotocol/oasis-wallet-web/blob/8be6fcb13735fa5dee04c8be75918078e5b6ef56/src/types/StringifiedBigInt.ts#L1-L3